client/lib/wpcom-undocumented/lib/site.js

```diff
@@ -183,6 +184,20 @@ UndocumentedSite.prototype.setOption = function( query, callback ) {
 	);
 }
 
+UndocumentedSite.prototype.postCounts = function( options, callback ) {
+	var type,
```

Comment:
We're not as concerned about mixed declaration types, so you can get away with ES6 (with non-top-of-function declaration) here if you'd prefer.